### DISCUSSION TITLE
CFP - Visualization Notes

### DISCUSSION THREAD

**Discussion by arky** [neocarto/bertin#86] [Ideas]
It would be great if Bertin.JS could be presented at upcoming PacificVis 2023 conference. What do you reckon @neocarto ?

> **CFP - Visualization Notes**
> 
> PacificVis 2023 features a short paper track, called “Visualization Notes”. The purpose of this track is to encourage young researchers to present their work and discuss with participants including present senior researchers. The submissions can be late-breaking results or work in progress, while they should be novel enough to attract interest from the visualization community. PacificVis 2023 will be held in Seoul, Korea on April 18-21, 2023. 
> 
> https://pvis2023.github.io/pvis2023/pages/cfps/cfp_notes
>

**Comment by neocarto** [comment]
What a great idea. Unfortunately, I'm not sure I'm available and have the budget for it. But if you go there, we can make a paper together that you present :-)

**Reply by arky** [reply]
Sounds like a plan. Let me jot down some ideas and reach out to you via email.

**Reply by neocarto** [reply]
Were you able to start thinking about it?

**Reply by arky** [reply]
Indeed, Will mail you a draft shortly.

**Reply by arky** [reply]
@neocarto the draft should be in your inbox. If not, give me a shout out.

**Comment by arky** [comment]
Really excited to announce that a paper has been submitted. Thank you @neocarto for all the help and support.

**Reply by neocarto** [reply]
Thanks especially to you. It's great.
